docs/turn-howto.md

```diff
@@ -129,26 +144,84 @@ This will install and start a systemd service called `coturn`.
+    56789 on the external IP must always be forwarded to port 56789 on the
+    internal IP). If your NAT server runs Linux (and you have administrative
+    rights to it), port forwarding may be applied with the following `iptables`
+    rules:
+
+    ```sh
+    # Enable forwarding from the NAT gateway to your subnet
+    # Based on https://wiki.debian.org/OpenVPN#Forward_traffic_to_provide_access_to_the_Internet
+    iptables -A FORWARD -i $EXTERNAL_INTERFACE_NAME -o $INTERNAL_INTERFACE_NAME -m state --state ESTABLISHED,RELATED -j ACCEPT
+    iptables  -A FORWARD -s $INTERNAL_INTERFACE_IPv4_SUBNET -o $EXTERNAL_INTERFACE_NAME -j ACCEPT
+    ip6tables -A FORWARD -s $INTERNAL_INTERFACE_IPv6_SUBNET -o $EXTERNAL_INTERFACE_NAME -j ACCEPT
+    iptables  -t nat -A POSTROUTING -s $INTERNAL_INTERFACE_IPv4_SUBNET -o $EXTERNAL_INTERFACE_NAME -j SNAT --to-source $EXTERNAL_NAT_IPv4_ADDRESS
+    ip6tables -t nat -A POSTROUTING -s $INTERNAL_INTERFACE_IPv6_SUBNET -o $EXTERNAL_INTERFACE_NAME -j SNAT --to-source $EXTERNAL_NAT_IPv6_ADDRESS
+
+    # forward coturn ports
+    iptables  -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p tcp -m multiport --dports 3478,3479,5349,5350 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv4_ADDRESS
+    iptables  -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p udp -m multiport --dports 3478,3479,5349,5350,49152:65535 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv4_ADDRESS
+    iptables  -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p tcp -m multiport --dports 5222,5269 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv4_ADDRESS
+
+    # same, but for ipv6
+    ip6tables -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p tcp -m multiport --dports 3478,3479,5349,5350 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv6_ADDRESS
+    ip6tables -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p udp -m multiport --dports 3478,3479,5349,5350,49152:65535 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv6_ADDRESS
+    ip6tables -t nat -A PREROUTING -i $EXTERNAL_INTERFACE_NAME -p tcp -m multiport --dports 5222,5269 -j DNAT --to-destination $INTERNAL_TURNSERVER_IPv6_ADDRESS
+    ```
+
+    For this to work, the NAT server must be configured to forward IPv4/IPv6
+    traffic. To check if forwarding is enabled, run
+    `cat /proc/sys/net/ipv4/ip_forward` and `cat /proc/sys/net/ipv6/conf/all/forwarding`
+    and confirm they both print `1`. If not, they may be set at runtime like so:
+
+    ```sh
+    echo 1 > /proc/sys/net/ipv4/ip_forward
+    echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
+    ```
+
+    To apply these settings persistently, add/uncomment these lines in
+    `/etc/sysctl.conf`, then run `sysctl -p`:
+
+    ```
+    net.ipv4.ip_forward=1
+    net.ipv6.conf.all.forwarding=1
+    ```
 
-    If you get this working, let us know!
+    (The `all` in any of the the IPv6 paths/settings mentioned above may be
+    substituted for the name of any specific network interface that must
+    forward traffic.)
```

Comment:
This is pretty long now. It might be a good idea to move it out to a separate section "configuring a firewall to forward traffic" or something?

Reply:
Good idea, I'll get to it.

Reply:
On second thought, I removed this entirely, as setting up your own NAT server isn't relevant for knowing how to get TURN working under a pre-existing NAT.